.github/workflows/test.yml

```diff
@@ -5,7 +5,7 @@ name: scvi-tools
 
 on:
   push:
-    branches: [master, 0.15.x, 0.16.x, 0.17.x]
+    branches: [master, scar, 0.15.x, 0.16.x, 0.17.x]
```

Comment:
no need for this

Reply:
just been using it because I was working on the `scar` instead of `master` branch in my repo. will remove it in time for the merge